sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/statsEstimation/BasicStatsEstimationSuite.scala

```diff
@@ -115,6 +119,16 @@ class BasicStatsEstimationSuite extends PlanTest with StatsEstimationTestBase {
       plan, expectedStatsCboOn = expectedCboStats, expectedStatsCboOff = expectedDefaultStats)
   }
 
+  test("command should report a dummy stats") {
```

Comment:
this test fails before this patch, because `ResolvedNamespace` doesn't implement `computeStats`, but it shouldn't do.